### PR TITLE
NodeGraph: Update no data message to show if no nodes

### PR DESCRIPTION
**Pull request by connorlindsey**
**What this PR does / why we need it**:
Adds some more error handling/info around service graphs. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by joey-grafana** [comment]
This looks good, I personally like the persistent alert as opposed to the notification.

This is timely as I'm thinking about the errors for InfluxDB. DS's seem to report if there was an error parsing the response or other HTTP errors and also more fine-grained errors related to the query itself. However, it feels like error reporting is done slightly differently amongst the various plugins. Perhaps we can take a more reusable approach in the future :)

**Review by joey-grafana — APPROVED** [review]

**Comment by aocenas** [comment]
What usecase does this cover? Because there is this message in query editor https://github.com/grafana/grafana/blob/1a7567d8c75f404554e191ae5fe1fc515fdfd5dd/public/app/plugins/datasource/tempo/QueryEditor/ServiceGraphSection.tsx#L32. Wouldn't this be duplication?

File: public/app/plugins/panel/nodeGraph/NodeGraph.tsx [hunk 1]
```diff
@@ -180,6 +180,8 @@ export function NodeGraph({ getLinks, dataFrames, nodeLimit }: Props) {
 
   const highlightId = useHighlight(focusedNodeId);
 
+  console.log(JSON.stringify(nodes, null, 2));
```
Comment (aocenas) [inline public/app/plugins/panel/nodeGraph/NodeGraph.tsx:183]: Missed console.log

**Review by aocenas — APPROVED** [review]
Looks good, just a forgotten console.log

**Pull request merged**